mneme-project/ambrosia/ambrosia-library/lib/src/webapp/skin/ambrosia.css

```diff
@@ -310,7 +310,10 @@ Sakai ~ itemAction
 	font-size: .8em;
 	white-space: nowrap;
 }
-
+.ambrosiaIcon
```

Comment:
Would it be better to define this centrally and then then use that class in the places it's needed?
```
.nonResponsive {
   max-width: none;
}
```